app/src/main/java/com/appdev/eateryblueandroid/ui/components/home/Main.kt

```diff
@@ -133,22 +185,77 @@ fun Main(
                         }
                     }
                 is MainItem.EaterySectionList ->
-                    EaterySectionPreview(
-                        eateries = eateries,
-                        section = item.section,
-                        selectSection = selectSection,
-                        selectEatery = selectEatery
-                    )
+                    Box(
+                        modifier = Modifier
+                            .animateItemPlacement()
+                            .padding(bottom = 12.dp)
+                    ) {
+                        EaterySectionPreview(
+                            eateries = eateries,
+                            section = item.section,
+                            selectSection = selectSection,
+                            selectEatery = selectEatery
+                        )
+                    }
                 is MainItem.EateryItem ->
                     Column(
-                        modifier = Modifier.padding(
-                            start = 16.dp,
-                            end = 16.dp,
-                            bottom = 12.dp
-                        )
+                        modifier = Modifier
+                            .padding(
+                                start = 16.dp,
+                                end = 16.dp,
+                                bottom = 12.dp
+                            )
+                            .animateItemPlacement()
                     ) {
                         EateryCard(eatery = item.eatery, selectEatery = selectEatery)
                     }
+                is MainItem.EaterySpacer ->
+                    Spacer(modifier = Modifier.height(12.dp))
+                is MainItem.EmptyMessage ->
+                    Column(
+                        modifier = Modifier
+                            .fillMaxWidth()
+                            .padding(top = ((screenHeight - 311 - 157) / 2).dp),
```

Comment:
Why these specific numbers? Are the alignments not enough?

Reply:
Oh, right. I resorted to putting specific numbers here because there's no really *good* way of simply making something take up max height in a LazyColumn. I tried multiple methods of making it take up the remaining screen height, but found that there really was not a good way. As such, I used this workaround.

Reply:
My next question is that is there a reason why Main.kt is framed as a LazyColumn instead of a regular Column with some if else to choose what to draw? The mainItems listOf is a little confusing

Reply:
Oh I think this was all here before 😭

Reply:
Yeah, that LazyColumn was there since day one of me working on Eatery Blue, lol. I'm assuming it has the same benefits of a RecyclerView over a LinearLayout.. however the mainItems blob is VERY difficult to read by consequence.